specification/e-referrals-service-api.yaml

```diff
@@ -299,10 +299,68 @@ paths:
           $ref: 'components/schemas/responses/CreateReferralErrorOutcome.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'
+  /STU3/v1/ReferralRequest/{referralRequestId}:
+    parameters:
+      - $ref: 'components/schemas/path/ReferralRequestId.yaml'
+    get:
+      security:
+        - bearerAuth: [ ]
+      description: |
+        ## Overview
+
+        Use this endpoint to retrieve details of a referral. This includes references to clinical attachments, related referrals and other important data.
+
+        ## Pre-requisites
+        In order to use this endpoint you must be an authenticated e-RS user and use one of the following e-RS roles:
+          - `REFERRING_CLINICIAN`
+          - `REFERRING_CLINICIAN_ADMIN`
+          - `SERVICE_PROVIDER_CLINICIAN`
+          - `SERVICE_PROVIDER_CLINICIAN_ADMIN`
```

Comment:
SERVICE_PROVIDER_CLINICIAN_ADMIN will have to be removed as we don't support it right now (even if the overview spec still has this in ) . I've raised this with Ba's already

Reply:
Removed from the yaml file. I see https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-54848 has been updated too, so we're aligned.